yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -64,6 +65,10 @@ var assessMigrationCmd = &cobra.Command{
 		validatePortRange()
 		validateSSLMode()
 		validateAssessmentMetadataDirFlag()
+		cmd.MarkFlagRequired("source-db-user")
```

Comment:
These Flags can't be marked as mandatory outright, because user can also pass `assessment-metadata-dir` , in which case user should not be allowed to pass these flags. https://yugabyte.atlassian.net/browse/DB-11188

We might have to validate the args manually...

Reply:
Oh yes, thanks for pointing it out. will check for that flag first.